kubelet/datadog_checks/kubelet/data/conf.yaml.example

```diff
@@ -2,6 +2,10 @@ init_config:
 
 instances:
 
+  ## WARNING: This check has no required parameter, if you intend to use any of the parameter
+  ## below, remove '{}' from the instance definition and use any parameter instead.
+  - {}
```

Comment:
why not revert to last known good state?

Reply:
https://github.com/DataDog/integrations-core/blob/69b6f2769cb1a84bbd3805557929d796353ba532/kubelet/datadog_checks/kubelet/data/conf.yaml.example

Reply:
https://github.com/DataDog/integrations-core/blob/b5c085b9e73f6ecf648bc018ab421fee9389899b/ecs_fargate/datadog_checks/ecs_fargate/data/conf.yaml.example#L5-L7

Reply:
Doing it elsewhere doesn't mean it works here. This broke because of the logic in the entrypoint, and we need to be sure that this works with the logic in the entrypoint.

Reply:
Confirmed it works